src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,87 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_LENGTH = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final int GROW_FACTOR = 2;
+    private int size;
+    private Node<K, V>[] map;
+
+    public MyHashMap() {
+        map = new Node[DEFAULT_LENGTH];
+    }
+
+    private static class Node<K, V> {
+        private final K key;
+        private V value;
+        private Node<K, V> next;
+
+        Node(K key, V value) {
+            this.key = key;
+            this.value = value;
+        }
+    }
 
     @Override
     public void put(K key, V value) {
+        Node<K, V> newNode = new Node<>(key, value);
+        int index = getNodeIndex(key);
+        if (size > map.length * DEFAULT_LOAD_FACTOR) {
+            resize();
+        }
```

Comment:
Check inside method resize and place it at start of method and you can create field threshold for checking load of map

Reply:
Not fixed